addons/payment_custom/models/payment_provider.py

```diff
@@ -77,3 +75,11 @@ def _get_removal_values(self):
         res = super()._get_removal_values()
         res['custom_mode'] = None
         return res
+
+    def _transfer_ensure_pending_msg_is_set(self):
+        transfer_providers_without_msg = self.filtered(
+            lambda p: p.code == 'custom' and p.custom_mode == "wire_transfer" and not p.pending_msg
+        )
+        if not transfer_providers_without_msg:
+            return
+        self.action_recompute_pending_msg(providers=transfer_providers_without_msg)
```

Comment:
Does `action_recompute_pending_msg` need to take a `providers` kwargs? I find it weird not to do `transfer_providers_without_msg.action_recompute_pending_msg()` instead. When called from the view, the action's `self` will be populated with the record.

Reply:
it is indeed better, thank you